ABOUT.md

```diff
@@ -0,0 +1,51 @@
+
+# What is MineColonies?
+Here, we would like to give you an overview of what our mod is.
+
+### Creativity 
+MineColonies is a town building mod that allows you create your own thriving Colony within Minecraft. It tends to depict a real-life scenario by providing you with different tools in your chest to aid your gaming experience. Featuring many NPC workers such as Builders, Crafters, Farmers, Guards, Enchanters and Alchemists, MineColonies provides the ability to create a rich and unique Colony.
+
+### Governance
+In MineColonies, you take control of what goes on in your Colony. Have an adequate number of villagers who will live in the town with you. Create a division of labour, hire workers to help in developing the Colony while you maturely act as a Supervisor.
+
+### Security and Replenishment
+The growth and safety of the colony solely depends on your governmental skills. Barbarians attack the colony on few nights. Hire guards, assign them weapons and armour to aid in protecting the Colony. As a Colony owner, you cannot let your villagers go hungry. You can always get food from your chest to feed the hungry villagers.
+
+### Have fun, Play with Friends
+Taking a dive into our multiplayer system is a good way to learn more about Colony building and management. A robust permission system in multi-player allows you to collaborate in a single Colony with your friends, or have individual colonies all competing to be the best!
+___
+##### MineColonies is a free and open source mod developed by LDT (Let's Dev Together), a non-profit community. The source code is on [GitHub.](https://github.com/ldtteam/minecolonies) Contribute to our Project. Found any bugs? Report an [issue](https://github.com/ldtteam/minecolonies/issues/new) so as to help us ensure you have the best gaming experience. For more information and enquiries, contact us on [Discord.](https://discord.gg/YEas2Yv)
+___
+Our developer team is continuously working to add more content to our mod and make the MineColonies experience even greater!
```

Comment:
I would add/write: "Our developers are a hard working well integrated coding team, continuously working to add more content to our mod and make the MineColonies experience even greater!"